### PR TITLE
Fix solo results' horizontal size on match song select

### PR DESCRIPTION
**Pull request by Joehuu**
Also fixes title on the header.

---

### CODE REVIEW

**Comment by peppy** [comment]
If not resolving an issue, please provide a screenshot.

**Comment by Joehuu** [comment]
Before:
osu_2019-06-01_07-43-19


After:
osu_2019-06-01_07-44-18

File: osu.Game/Screens/Play/MatchSoloResults.cs [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+// See the LICENCE file in the repository root for full licence text.
+
+using Humanizer;
+using osu.Framework.Graphics;
+using osu.Game.Scoring;
+using osu.Game.Screens.Multi;
+
+namespace osu.Game.Screens.Play
+{
+    public class MatchSoloResults : SoloResults, IMultiplayerSubScreen
```
Comment (peppy) [inline osu.Game/Screens/Play/MatchSoloResults.cs]: This should probably go in the multiplayer namespace.

File: osu.Game/Screens/Select/SongSelect.cs [hunk 2]
```diff
@@ -574,6 +574,8 @@ protected virtual void UpdateBeatmap(WorkingBeatmap beatmap)
                 beatmap.Track.Looping = true;
         }
 
+        public virtual void ShowResults() => BeatmapDetails.Leaderboard.ScoreSelected += s => this.Push(new SoloResults(s));
```
Comment (smoogipoo) [inline osu.Game/Screens/Select/SongSelect.cs]: Keep it as it was before, but add/use a `CreateResultsScreen(ScoreInfo score)` method.

**Review by smoogipoo — CHANGES_REQUESTED** [review]

**Comment by Joehuu** [comment]
Closing as the original issue this PR was trying to resolve was fixed after the `BackButton` changes. Code quality that was done here may be done with the redesign.

**Pull request closed without merging**